src/include/resource.h

```diff
@@ -104,6 +106,7 @@ typedef struct resource_def {
 	unsigned int rs_type : ATRDTYPE;  /* type of resource,see attribute.h */
 	unsigned int rs_entlimflg;	  /* tracking entity limits for this  */
 	struct resource_def *rs_next;
+	unsigned int rs_custom; /* bit flag to indicate custom resource or builtin */
```

Comment:
shall we align the comment.

Reply:
these are all "aligned" by the "clang-formatter"  - the guideline is that are no longer required (actually were never required) to align variables into a column